### ISSUE TITLE
The following classes could not be instantiated......Tip: Use View.isInEditMode() in your custom views to skip code when shown in Eclipse?

### ISSUE THREAD

**Issue by WaylanPunch** [hardik-trivedi/FloatingLabel#5]
The following classes could not be instantiated:

See the Error Log (Window > Show View) for more details.

**Tip: Use View.isInEditMode() in your custom views to skip code when shown in Eclipse**

android.content.res.Resources$NotFoundException: Could not find dimen resource matching value 0x7F040001 (resolved name: activity_vertical_margin) in current configuration.

Need set a code snippet into the constructor method:

```
if (!isInEditMode()) {
         createLayout(null);
}
```

**Comment by hardik-webonise** [comment]
@WaylanPunch Yes you are right. I will do that. You can clone the code and contribute. I also need to provide gradle support.
